zeebe/broker/src/main/java/io/camunda/zeebe/broker/system/partitions/impl/PartitionProcessingState.java

```diff
@@ -69,35 +75,61 @@ public boolean shouldProcess() {
   }
 
   public boolean isExportingPaused() {
-    return isExportingPaused;
+    return exporterState.equals(PAUSED);
   }
 
   @SuppressWarnings({"squid:S899"})
-  /** Returns true if exporting is paused */
-  public boolean pauseExporting() throws IOException {
-    final File persistedExporterPauseState =
-        getPersistedPauseState(PERSISTED_EXPORTER_PAUSE_STATE_FILENAME);
-    persistedExporterPauseState.createNewFile();
-    if (persistedExporterPauseState.exists()) {
-      isExportingPaused = true;
-      return true;
+  /** Returns true if exporting is paused. This method overrides the effects of soft pause. */
+  public boolean pauseExporting() {
+    try {
+      setPersistedExporterState(PAUSED);
+    } catch (final IOException e) {
+      return false;
+    }
+    return true;
+  }
+
+  /** Returns true if soft exporting is paused. This method overrides the effects of hard pause. */
+  public boolean softPauseExporting() {
+    try {
+      setPersistedExporterState(SOFT_PAUSED);
+    } catch (final IOException e) {
+      return false;
     }
-    return false;
+    return true;
   }
 
-  /** Returns true if exporting is resumed */
-  public boolean resumeExporting() throws IOException {
+  /** Returns true if exporting is resumed. This method resumes both soft and "hard" exporting. */
+  public boolean resumeExporting() {
+    try {
+      setPersistedExporterState(EXPORTING);
```

Comment:
:wrench: I don't think we need to set the "exporting" state in the file. This is the normal state. We could as well just delete the file.